core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -19,71 +19,64 @@
 
 var waitFor = require('./waitFor.js');
 var action = require('./action.js');
+const { element } = require('protractor');
```

Comment:
I don't think you need this. The other protractor utility files don't have this import

Reply:
Done